README.md

```diff
@@ -2,7 +2,7 @@
 # Leaderboard
```

Comment:
- [ ] Please add a better description of how to set up the project when you clone the repository, you should add instructions, as if you were explaining it to someone who does not know anything about code. 🤓 
    - How do I install the necessary packages?
    - How do I run the project locally?